OWNERS_ALIASES

```diff
@@ -30,7 +30,6 @@ aliases:
     - keithmattix
 
   gateway-api-conformance-reviewers:
-    - arkodg
```

Comment:
Changed while I was in here because I believe wince approver is a superset of reviewer, this was redundant (and maybe would unfairly distribute extra review responsibility to Arko?)